src/drivers/serac.cpp

```diff
@@ -98,23 +99,23 @@ int main(int argc, char* argv[])
 
   // Check for the doc creation command line argument
   bool create_input_file_docs = cli_opts.find("create_input_file_docs") != cli_opts.end();
+  // Check for the output fields command line argument
+  bool output_fields = cli_opts.find("output_fields") != cli_opts.end();
 
   // Create DataStore
   axom::sidre::DataStore datastore;
 
   // Intialize MFEMSidreDataCollection
   // If restart_cycle is non-empty, then this is a restart run and the data will be loaded here
-  serac::StateManager::initialize(datastore, restart_cycle);
+  serac::StateManager::initialize(datastore, "serac", restart_cycle);
 
   // Initialize Inlet and read input file
   auto inlet = serac::input::initialize(datastore, input_file_path);
   serac::defineInputFileSchema(inlet);
 
   // Optionally, create input file documentation and quit
   if (create_input_file_docs) {
-    auto writer = std::make_unique<axom::inlet::SphinxWriter>("serac_input.rst");
-    inlet.registerWriter(std::move(writer));
-    inlet.write();
+    inlet.write(axom::inlet::SphinxWriter("serac_input.rst"));
```

Comment:
Ooo! Much nicer!

Reply:
@joshessman-llnl 's work! now you can have multiple writers.. all the writers all the time!